include/ddc/transform_reduce.hpp

```diff
@@ -29,18 +33,22 @@ inline T transform_reduce(
     return init;
 }
 
-/** A parallel reduction over a 2d domain
- * @param[in] domain    the 2d domain to iterate
- * @param[in] reduce    a reduction operation
- * @param[in] transform a functor taking a 2d discrete coordinate as parameter
+/** A sequential reduction over a 2D domain
+ * @param[in] policy the execution policy to use
+ * @param[in] domain the range over which to apply the algorithm
+ * @param[in] init the initial value of the generalized sum
+ * @param[in] reduce a binary FunctionObject that will be applied in unspecified order to the
+ *            results of transform, the results of other reduce and init. 
+ * @param[in] transform a unary FunctionObject that will be applied to each element of the input
+ *            range. The return type must be acceptable as input to reduce
  */
-template <class DDim1, class DDim2, class T, class BinaryReductionOp, class BinaryTransformOp>
+template <class DDim1, class DDim2, class T, class BinaryReductionOp, class UnaryTransformOp>
 inline T transform_reduce(
-        serial_policy,
+        serial_policy policy,
```

Comment:
We will have an "unused warning", shall we move the documentation to a declaration without definition ?

Reply:
or use [`maybe_unused`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)

Reply:
```suggestion
        [[maybe_unused]] serial_policy policy,
```

Reply:
Why not if it does not appear on the documentation